exchanges/bybit/bybit_test.go

```diff
@@ -0,0 +1,1850 @@
+package bybit
+
+import (
+	"context"
+	"errors"
+	"log"
+	"os"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/config"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/sharedtestvalues"
+)
+
+// Please supply your own keys here to do authenticated endpoint testing
+const (
+	apiKey    = ""
+	apiSecret = ""
```

Comment:
It would be great if you could also add in the field `canManipulateRealOrders = false` like with other exchange tests. We use `canManipulateRealOrders` to prevent orders from being impacted if someone runs tests and happens to have the API key filled. This flag means it will only run when explicitly changed

Reply:
Alright. So we add this check to all the POST Auth. API Right? @gloriousCode

Reply:
DELETE and POST, thanks! 👍